lib/metrics/gc.js

```diff
@@ -6,8 +6,10 @@ let perf_hooks;
 try {
 	// eslint-disable-next-line
 	perf_hooks = require('perf_hooks');
+	new perf_hooks.PerformanceObserver();
```

Comment:
we don't need to close this in some way? i.e. https://nodejs.org/api/perf_hooks.html#performanceobserverdisconnect